Web/fluid_properties/Validation/NelsonValidation.py

```
@@ -1,40 +1,45 @@
+# CoolProp v8; bd CoolProp-r9sq.27.)  The ``h`` column below is printed in
+# kJ/kg_da, so the SI enthalpy is divided by 1000 on the way out.
 
 print("Validation against H.F. Nelson and H.J. Sauer,\"Formulation for High-Temperature Properties for Moist Air\", HVAC&R Research v.8 #3, 2002")
 print("Note: More accurate formulation employed than in Nelson.  Just for sanity checking")
 print("Yields a negative relative humidity for Tdb=5C,Twb=-3C, point omitted")
 tdb = [5, 5, 5, 25, 25, 25, 25, 50, 50, 50, 50, 50, 50, 50]
 twb = [5, 2, -1, 25, 20, 15, 10, 50, 40, 30, 25, 22, 20, 19]
+
+
+def _table(P):
+    print("{Tdb:10s}{Twb:10s}{Tdp:10s}{R:10s}{W:10s}{h:10s}{v:10s}".format(W='W', Twb='Twb', Tdp='Tdp', Tdb='Tdb', v='v', h='h', s='s', R='RH'))
```

Ruff (0.15.15)

[error] `.format` call has unused named argument(s): s

Remove extra named arguments: s

(F522)